core/src/main/scala/org/genivi/sota/core/DeviceUpdatesResource.scala

```diff
@@ -65,9 +66,15 @@ class DeviceUpdatesResource(db: Database,
 
   protected lazy val updateService = new UpdateService(DefaultUpdateNotifier, deviceRegistry)
 
+  private def sendDeviceSeen(id: Uuid): Future[Unit] = {
+    async {
+      val namespace = await(deviceRegistry.fetchMyDevice(id).map(_.namespace))
```

Comment:
Hmm so this is what is happening
1) Core contacts device registry to get namespace
2) Core publishes event
3) Device registry register events

So what was the point of this change?